client/lib/feed-stream-store/index.js

```diff
@@ -228,7 +242,15 @@ function feedStoreFactory( storeId ) {
 		} );
 	} else if ( storeId === 'recommendations_posts' ) {
 		store = getStoreForRecommendedPosts( storeId );
-	} else if ( storeId === 'coldstart_posts' ) {
+	} else if ( storeId === 'cold_posts' ) {
```

Comment:
maybe just a `else if ( startsWith( storeId, 'cold_posts' ) )` to kill some of the duplication here?
where you also have `import { startsWith } from 'lodash'` in the import section